src/sdpLoops.jl

```diff
@@ -159,9 +159,10 @@ function sdp_u_w_loop(sampling_size::Int, samples::Array,
 
                 count_admissible_w = count_admissible_w + proba
 
-                ind_next_state = real_index_from_variable(next_state, x_bounds,
-                                                            x_steps)
-                next_V = Vitp[ind_next_state...]
+                #ind_next_state = real_index_from_variable(next_state, x_bounds, x_steps)
+                ind_next_state = index_from_variable(next_state, x_bounds,x_steps)
+                #next_V = Vitp[ind_next_state...]
```

Comment:
dead comment?

Reply:
real_index_from_variable return real number, so cannot to use of index.
index_from_variable return integer index